### PR TITLE
cksfv: update to 1.3.15

### PR DESCRIPTION
**Pull request by clyhtsuriva**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64
- I built this PR locally for these architectures :
  - aarch64-musl
  - armv7l
  - armv6l-musl

### CODE REVIEW

**Comment by clyhtsuriva** [comment]
CI/CD issue :
```
Run common/travis/fetch_upstream.sh
  common/travis/fetch_upstream.sh
  shell: /bin/bash -e {0}
  env:
    PATH: /usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin:/usr/local/bin:/tmp/bin
    XLINT: 1
    LICENSE_LIST: common/travis/license.lst
Fetching upstream...
fatal: remote error: 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
Error: Process completed with exit code 128.
```

File: srcpkgs/cksfv/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 pkgname=cksfv
-version=1.3.14
-revision=3
+version=1.3.15
+revision=1
 build_style=configure
 short_desc="SFV checksum utility (simple file verification)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="http://zakalwe.fi/~shd/foss/cksfv/"
```
Comment (paper42) [inline srcpkgs/cksfv/template:9]:
```suggestion
homepage="https://zakalwe.fi/~shd/foss/cksfv/"
```

File: srcpkgs/cksfv/template [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-distfiles="http://zakalwe.fi/~shd/foss/cksfv/files/${pkgname}-${version}.tar.gz"
-checksum=010facce85b317b7b9f952c84cd59ce03476a2b685db33a49b6ea0d1b38378ea
+distfiles="http://zakalwe.fi/~shd/foss/cksfv/files/${pkgname}-${version}.tar.bz2"
```
Comment (paper42) [inline srcpkgs/cksfv/template:10]:
```suggestion
distfiles="https://zakalwe.fi/~shd/foss/cksfv/files/cksfv-${version}.tar.bz2"
```

File: srcpkgs/cksfv/template [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'cksfv'
 pkgname=cksfv
-version=1.3.14
-revision=3
+version=1.3.15
+revision=1
 build_style=configure
```
Comment (paper42) [inline srcpkgs/cksfv/template:5]:
```suggestion
build_style=gnu-configure
```

File: srcpkgs/cksfv/template [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="http://zakalwe.fi/~shd/foss/cksfv/"
-distfiles="http://zakalwe.fi/~shd/foss/cksfv/files/${pkgname}-${version}.tar.gz"
-checksum=010facce85b317b7b9f952c84cd59ce03476a2b685db33a49b6ea0d1b38378ea
+distfiles="http://zakalwe.fi/~shd/foss/cksfv/files/${pkgname}-${version}.tar.bz2"
+checksum=a173be5b6519e19169b6bb0b8a8530f04303fe3b17706927b9bd58461256064c
 
 pre_configure() {
 	configure_args="--prefix=/usr --package-prefix=${DESTDIR} --bindir=/usr/bin --mandir=/usr/share/man"
```
Comment (paper42) [inline srcpkgs/cksfv/template:14]:
```suggestion
	configure_args+="--package-prefix=${DESTDIR}"
```

**Comment by clyhtsuriva** [comment]
Going to rebase..

**Comment by paper42** [comment]
Thanks!

**Pull request merged**